### ISSUE TITLE
An error breaks the installation of pandoc-pyplot

### ISSUE THREAD

**Issue by cxzchange** [LaurentRDC/pandoc-pyplot#1]
when i install pandoc-pyplot with `cabal install`, i catch an error:
`
[1 of 3] Compiling Text.Pandoc.Filter.Scripting ( src/Text/Pandoc/Filter/Scripting.hs, dist/build/Text/Pandoc/Filter/Scripting.o )

src/Text/Pandoc/Filter/Scripting.hs:45:50: error:
    * Variable not in scope:
        (<>) :: m0 ExitCode -> String -> IO ExitCode
    * Perhaps you meant one of these:
        `</>' (imported from System.FilePath),
        `<$>' (imported from Prelude), `*>' (imported from Prelude)
      Perhaps you want to add `<>' to the import list in the import of
      `Data.Monoid' (src/Text/Pandoc/Filter/Scripting.hs:27:1-37).

src/Text/Pandoc/Filter/Scripting.hs:57:32: error:
    * Variable not in scope: (<>) :: [Char] -> String -> t0
    * Perhaps you meant one of these:
        `</>' (imported from System.FilePath),
        `<$>' (imported from Prelude), `*>' (imported from Prelude)
      Perhaps you want to add `<>' to the import list in the import of
      `Data.Monoid' (src/Text/Pandoc/Filter/Scripting.hs:27:1-37).

src/Text/Pandoc/Filter/Scripting.hs:57:46: error:
    * Variable not in scope: (<>) :: t0 -> [Char] -> PythonScript
    * Perhaps you meant one of these:
        `</>' (imported from System.FilePath),
        `<$>' (imported from Prelude), `*>' (imported from Prelude)
      Perhaps you want to add `<>' to the import list in the import of
      `Data.Monoid' (src/Text/Pandoc/Filter/Scripting.hs:27:1-37).
cabal: Leaving directory '.'
cabal: Error: some packages failed to install:
pandoc-pyplot-1.0.2.0 failed during the building phase. The exception was:
ExitFailure 1
`

on Ubuntu 18.04 gcc-7 ghc-8.0.2 cabal-install version 1.24.0.2

**Comment by LaurentRDC** [comment]
Hi @cxzchange ,

The only thing I can think of is that your version of the `base` library is < 4.9. The monoid mappend operator `<>` was added in the Prelude in version 4.9 ([see here](https://hackage.haskell.org/package/base-4.12.0.0/docs/Prelude.html#g:9)). However, I mistakenly allowed older versions of `base`.

As of right now, I have edited the package metadata on Hackage ([see here](http://hackage.haskell.org/package/pandoc-pyplot)) so that this particular error will not occur: minimum version of `base` is now 4.9. If you want to try again, please update your list of cabal packages before installing:

```sh
cabal update # Make cabal see the revised version of pandoc-pyplot's metadata
cabal install pandoc-pyplot
```

Let me know if this fixes your problem. 

I will release a new version of `pandoc-pyplot` that allows for older `base` versions as soon as I can.

**Comment by LaurentRDC** [comment]
I've released a new version (1.0.3.0) which should fix this. @cxzchange let me know if I can close this issue.

**Comment by cxzchange** [comment]
@LaurentRDC I'm sorry for replying you so late! Your work is effective for me. Thank you very much!
